kura/org.eclipse.kura.easse/build.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2017 Eurotech and/or its affiliates and others
```

Comment:
I thought we agreed upon keeping this in sync. So either both times "Amit Kumar Mondal" or "Eurotech".